### ISSUE TITLE
Program type already present: org.openqa.selenium.SearchContext

### ISSUE THREAD

**Issue by emptywalker** [appium/java-client#1141]
## Description

I encapsulated java-client in a jar, and the following error occurred when I referenced the jar in my project
```
Error: Program type already present: org.openqa.selenium.SearchContext
```
I find where the SearchContext is defined：
* first:
![image](https://user-images.githubusercontent.com/12522809/57268632-1e3ab200-70b7-11e9-9637-b8701a76507d.png)
![image](https://user-images.githubusercontent.com/12522809/57268648-2a267400-70b7-11e9-9d5f-a1a84a974ca6.png)
* second:
![image](https://user-images.githubusercontent.com/12522809/57268681-462a1580-70b7-11e9-8ca2-1e39cfc8df16.png)

They have the same package name and class name, so there is a conflict

## Environment

* java client build version @7.0.0

* app/build.gradle
```
implementation 'io.appium:java-client:7.0.0'
```

## demand
how to fix this?

**Comment by kool79** [comment]
#1021

**Comment by Amrzizo** [comment]
Dears, any help here!

**Comment by valfirst** [comment]
Fixed by #1555
